deployment/helm/node-feature-discovery/templates/master.yaml

```
@@ -82,6 +82,9 @@ spec:
             - "-port={{ .Values.master.port | default "8080" }}"
             {{- if .Values.enableNodeFeatureApi }}
             - "-enable-nodefeature-api"
+            {{- if  gt (float64 .Values.master.replicaCount) 1.0 }}
```

Replicacount is an integer so I think we could simplify here a bit
```suggestion
            {{- if  gt Values.master.replicaCount 1 }}
```

---

I tried it but it seems that Helm is reading yaml numbers as floats:
```sh
$ helm template nfd ./node-feature-discovery --namespace node-feature-discovery --set enableNodeFeatureApi=true
Error: template: node-feature-discovery/templates/master.yaml:85:20: executing "node-feature-discovery/templates/master.yaml" at <gt .Values.master.replicaCount 1>: error calling gt: incompatible types for comparison

Use --debug flag to render out invalid YAML
```

interesting as this works fine:
```sh
$ helm template nfd ./node-feature-discovery --namespace node-feature-discovery --set enableNodeFeatureApi=true,master.replicaCount=1
```

---

Hmm, interesting. I needed to try it out myself and yes, you're right. An evolved suggestion:

```suggestion
            {{- if  gt (int .Values.master.replicaCount) 1 }}
```